### PR TITLE
Review base stability

### PR DESCRIPTION
**Pull request by mnsnor**

### CODE REVIEW

File: report/finished/chapters/BASE_stability.tex [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+\begin{equation}
+    G_{ol}(s)=\frac{1}{2 r_b s}
+\end{equation}
+
+\begin{equation}
+    G_{cl}(s)=\frac{G_{ol}(s)}{1+G_{ol}(s)}=\frac{(2r_b s)}{(2r_b s)*(2r_b s + 1)}
+\end{equation}
+
+\noindent The function has a pole and zero cancellation at 0. In Fig.\ref{fig:step1} the step response, poles and zeros are plotted for the closed loop system. Since all the poles appeare on the negative real plane, the system is stable.
```
Comment (edwced) [inline report/finished/chapters/BASE_stability.tex]: = zero

Comment (edwced) [reply report/finished/chapters/BASE_stability.tex]: we might want to have an reference on why this makes it stable

File: report/finished/chapters/BASE_stability.tex [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+\end{figure}
+
+\noindent The \textbf{PD}-controller discussed in previous sections only need to improve the system response, since the overall system is already stable. Introducing the controller to the system gives us the closed loop function:
+
+\begin{equation}
+    G_{clPD} = \frac{2r_b s(k_p + k_d s)}{2r_b s(s(2r_b+k_d)+k_p)}
+\end{equation}
+
+\noindent From this it's easy to see that the system will remain stables for $k_d > - 2 r_b$ and that the value of $k_d$ won't have any effect on the stability of the system in continuous time.\\
+\indent The values for $k_p$ and $k_d$ is therefore chosen in such a manner that $ 0 > k_d > -2 r_b$ and $k_p$ so that the greatest error input from the \textbf{MV}-node will not saturate the actuators. The result of these are shown in Fig.\ref{fig:step2} and a comparison was made in table \ref{tab:with_without}
```
Comment (edwced) [inline report/finished/chapters/BASE_stability.tex]: was made, represented in table

File: report/finished/chapters/BASE_stability.tex [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width= 0.7\textwidth]{img/closed_loop_step_PD.eps}
+    \caption{The closed loop system with a controller}
```
Comment (edwced) [inline report/finished/chapters/BASE_stability.tex]: . end of caption, if we do that.

**Review by edwced — COMMENTED** [review]
nice !

File: report/finished/chapters/BASE_stability.tex [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+The $\Delta v$ will be seen as a constant and will be ignored in future equations. Using \eqref{eq:stab1}, the open-loop and closed-loop equation becomes:
+
+\begin{equation}
+    G_{ol}(s)=\frac{1}{2 r_b s}
+\end{equation}
+
+\begin{equation}
+    G_{cl}(s)=\frac{G_{ol}(s)}{1+G_{ol}(s)}=\frac{(2r_b s)}{(2r_b s)*(2r_b s + 1)}
+\end{equation}
```
Comment (NiklasDahlquist1) [inline report/finished/chapters/BASE_stability.tex:20]: End sentence with a period (.)

File: report/finished/chapters/BASE_stability.tex (hunk 1 above)
Comment (NiklasDahlquist1) [reply report/finished/chapters/BASE_stability.tex]: what function? Reference it.

File: report/finished/chapters/BASE_stability.tex [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    \centering
+    \includegraphics[width=0.7\textwidth]{img/closed_loop_step.eps}
+    \caption{The closed loop system without a controller}
+    \label{fig:step1}
+\end{figure}
+
+\noindent The \textbf{PD}-controller discussed in previous sections only need to improve the system response, since the overall system is already stable. Introducing the controller to the system gives us the closed loop function:
+
+\begin{equation}
+    G_{clPD} = \frac{2r_b s(k_p + k_d s)}{2r_b s(s(2r_b+k_d)+k_p)}
```
Comment (NiklasDahlquist1) [inline report/finished/chapters/BASE_stability.tex:34]: End sentence with a period (.)

File: report/finished/chapters/BASE_stability.tex [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|ccc|}
+        \hline & Without controller & \textbf{PD}-controller & Discrete Time \textbf{PD}-controller \\
+        \hline Rise Time & 0.8788$s$ & 0.0879$s$ & 0.1000$s$ \\ 
+        Settling Time & 1.5648$s$ & 0.1565$s$ & 0.2500$s$ \\
+        \hline
+    \end{tabular}
+    \caption{Comparison between the closed loop system}
```
Comment (NiklasDahlquist1) [inline report/finished/chapters/BASE_stability.tex]: I think the caption should be at the top of a table?

File: report/finished/chapters/BASE_stability.tex [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    \label{tab:with_without}
+\end{table}
+
+
+\subsubsection{Discrete Time}
+Using MATLAB to move from continuous time to discrete time with a update frequency of 20 Hz\footnote{The update frequency of the camera} and an input delay of $1/30s$\footnote{Estimated input delay to the actuators}. In Fig.\ref{fig:stepD} it's clear that the pole is within the unit circle on the discrete time plane and that the responds follows the step input. In conclusion, the system is stable using the $k_p$ and $k_d$ values proposed above, even in discrete time.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width= 0.7\textwidth]{img/closed_loop_step_D.eps}
```
Comment (NiklasDahlquist1) [inline report/finished/chapters/BASE_stability.tex:65]: Add legends to the plot.

File: report/finished/chapters/BASE_stability.tex [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+    \caption{The discrete step response and the poles and zeros of the discrete transfer function}
```
Comment (NiklasDahlquist1) [inline report/finished/chapters/BASE_stability.tex]: Add that the continuous response is included in the figure.

**Review by NiklasDahlquist1 — COMMENTED** [review]
Add periods to the captions

**Pull request merged**